python/sidecar/http.py

```diff
@@ -0,0 +1,85 @@
+from fastapi import FastAPI
+
+import sidecar
+from sidecar.typing import (
+    ChatRequest,
+    ChatResponse,
+    DeleteRequest,
+    DeleteStatusResponse,
+    IngestRequest,
+    IngestResponse,
+    IngestStatusResponse,
+    ReferenceUpdate,
+    RewriteRequest,
+    RewriteResponse,
+    SearchRequest,
+    SearchResponse,
+    TextCompletionRequest,
+    TextCompletionResponse,
+    UpdateStatusResponse,
+)
+
+app = FastAPI()
+v0 = FastAPI()
+
+
+@v0.get("/")
+async def index():
+    return {"message": "Hello World"}
+
+
+@v0.post("/ingest/upload")
+async def upload() -> IngestResponse:
+    # response = sidecar.ingest.run_ingest()
+    raise NotImplementedError("Upload is not yet implemented")
+
+
+@v0.get("/ingest/status")
+async def get_statuses() -> IngestStatusResponse:
+    response = sidecar.ingest.get_statuses()
+    return response
```

Comment:
any reason to keep the `response` variable here instead of `return sidecar.ingest.get_statuses()`? (also below)

Reply:
Nah, just stylistic and consistency with the other functions